gems/aws-sdk-core/lib/aws-sdk-core/ecs_credentials.rb

```diff
@@ -39,6 +39,10 @@ class Non200Response < RuntimeError; end
     # @option options [Integer] :port (80)
     # @option options [String] :credential_path By default, the value of the
     #   AWS_CONTAINER_CREDENTIALS_RELATIVE_URI environment variable.
+    # @option options [String] :endpoint The ECS credential endpoint.
+    #   By default, this is the value of the AWS_CONTAINER_CREDENTIALS_FULL_URI
+    #   environment variable. This value is ignored if `credential_path` or
+    #   ENV['AWS_CONTAINER_CREDENTIALS_RELATIVE_URI'] is set.
```

Comment:
May want to update docs to mention that port and ip address are ignored when endpoint is used (ie, when it is set and when the credential path is not).